packages/teams-js/src/internal/utils.ts

```diff
@@ -429,6 +429,12 @@ export function validateUrl(url: URL, errorToThrow?: Error): void {
   }
 }
 
+export function fullyQualifyUrlString(urlString: string): URL {
+  const link = document.createElement('a');
+  link.href = urlString;
```

Comment:
FWIW, I recently learned that `link.toString()` returns a read-only version of `link.href` -- not actually sure if there's any valid reason to prefer it in this case, but thought I'd share my recent awareness.

Reply:
That's interesting! I think we still want href here since we do want to overwrite href with a new value, but still might be useful someday.

Reply:
Oh duh. Yes of course!